### PR TITLE
Update/add hs tooltips TM-2280 TM-2277

### PR DESCRIPTION
**Pull request by scott062**
- Tooltip added

- Orange expiration Icon added once HS has expired

### CODE REVIEW

File: src/sass/_handshakeStatus.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
     background-color: $color-red;
   }
 
+  .expired {
+    background-color: #F7B578;
```
Comment (elizabeth-jimenez) [inline src/sass/_handshakeStatus.scss:25]: can we make this a variable?

File: src/sass/_handshakeStatus.scss [hunk 2]
```diff
@@ -32,4 +36,46 @@
...
+        padding-right: 10px;
+        border-right: 1px solid $color-gray-light;
+      }
+      div:nth-child(2) {
+        padding-left: 10px;
+      }
+      .title {
+        font-size: 16px;
+        font-weight: bold;
+        // align-self: center;
```
Comment (elizabeth-jimenez) [inline src/sass/_handshakeStatus.scss:72]: 😭 🧼

File: src/sass/_handshakeStatus.scss (hunk 1 above)
Comment (scott062) [reply src/sass/_handshakeStatus.scss:25]: yes :)

Comment (scott062) [reply src/sass/_handshakeStatus.scss]: Done!

File: src/Components/Handshake/HandshakeStatus/HandshakeStatus.jsx [hunk 3]
```diff
@@ -9,51 +12,108 @@ const HandshakeStatus = props => {
...
+            </div>
+          </div>
+        }
+        theme="hs-status"
+        arrow
+        tabIndex="0"
+        interactive
+        useContext
+      >
+        <div className="hs-status-container">
```
Comment (mjoyce91) [inline src/Components/Handshake/HandshakeStatus/HandshakeStatus.jsx:99]: Could add `cursor: help` to this class

**Comment by mjoyce91** [comment]
Bidder accepted handshake and is registered, but we're before the expiration
Screen Shot 2021-07-09 at 9 38 40 AM

Bidder accepted handshake and is registered, but we're after the expiration
Screen Shot 2021-07-09 at 9 39 09 AM

Expected behavior?

**Comment by scott062** [comment]
> Bidder accepted handshake and is registered, but we're before the expiration
> Screen Shot 2021-07-09 at 9 38 40 AM
> 
> Bidder accepted handshake and is registered, but we're after the expiration
> Screen Shot 2021-07-09 at 9 39 09 AM
> 
> Expected behavior?

No lol. I think i have the fix for it

File: src/Components/Handshake/HandshakeStatus/HandshakeStatus.jsx [hunk 4]
```diff
@@ -102,7 +102,7 @@ const HandshakeStatus = props => {
           </div>
           <div className={`hs-status-bidder ${bidderStyle.bidder}`}>
             {
-              isExpired ?
+              bidderStyle.bidderIcon === 'clock-o' ?
```
Comment (mjoyce91) [inline src/Components/Handshake/HandshakeStatus/HandshakeStatus.jsx]: This is fine for now, but let's update to something less brittle (what if we decide to change the icon in the future?)

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**